operations/mimir-mixin/dashboards/reads.libsonnet

```diff
@@ -441,5 +441,42 @@ local filename = 'mimir-reads.json';
     // Object store metrics for the querier.
     .addRows(
       $.getObjectStoreRows('Blocks object store (querier accesses)', 'querier')
+    )
+    .addRowIf(
+      $._config.show_reactive_limiter_panels,
+      $.row('Instance Limits')
+      .addPanel(
+        $.timeseriesPanel('Ingester per %s blocked requests' % $._config.per_instance_label) +
+        $.hiddenLegendQueryPanel(
+          'sum by (%s) (cortex_ingester_reactive_limiter_blocked_requests{%s, request_type="read"})'
```

Comment:
These metric names don't seem right, are they already rates? We should be emitting counters and computing a rate at query time.

Reply:
In that case, the current names are fine. I was just confused because they seemed like they were counters.